spec/operators/skipUntil-spec.ts

```diff
@@ -218,12 +206,12 @@ describe('Observable.prototype.skipUntil', () => {
     expectSubscriptions(skip.subscriptions).toBe(skipSubs);
   });
 
-  it('should skip all element when another observable unsubscribed early before emit', () => {
+  it('should skip skip all elements if notifier is unsubscribed explicitly before the notifier emits', () => {
```

Comment:
Is this test really required? To me, it looks like it only tests that calling [`unsubscribe`](https://github.com/ReactiveX/rxjs/blob/6.0.0-rc.0/src/internal/Subject.ts#L97-L101) on the `Subject` sets its `closed` property to `true`.

Reply:
Yeah, this is one weird case where semantics are a little different. In almost every other case, the notifier can only complete, error or last forever. This is a case where it can stop notifying and not still be active.